### DISCUSSION TITLE
蓝奏云-用RCLONE挂载ALIST为本地文件夹，不能上传文件，另外，蓝奏云不能在线播放，

### DISCUSSION THREAD

**Discussion by goldsoft8888** [AlistGo/alist#5528] [General]
蓝奏云-用RCLONE挂载ALIST为本地文件夹，不能上传文件。用ALIST直接上传蓝奏云盘是正常的， 用RCLONE+ALIST挂载其它网盘，上传也是正常的。另外，蓝奏云盘不能在线播放，

**Comment by foxxorcat** [comment]
1. 如果要用rclone需要加选项 --ignore-size 上传后不去检测大小，alist后台蓝奏云存储设置里面打开修复文件信息。
2. 蓝奏云盘官方不支持上传视频

**Reply by goldsoft8888** [reply]
谢谢，

**Comment by goldsoft8888** [comment]
{"code":500,"message":"failed link: failed get link: failed get link: not find file page param","data":null}

文件下载不了，提示

**Comment by foxxorcat** [comment]
#5332

**Reply by goldsoft8888** [reply]
非常感谢，解决了
